src/lib.rs

```diff
@@ -48,6 +48,8 @@ bitflags! {
     #[repr(transparent)]
     #[derive(Copy, Clone, Debug, Eq, PartialEq)]
     struct ReceiveStatus: u16 {
+        /// No error.
+        const NE = 0;
```

Comment:
This isn't a bit, so I think it won't work well with the `bitflags` macro. If you want a normal associated constant you can do it in a separate `impl` block for `ReceiveStatus` outside of the macro, or you can just use `ReceiveStatus::empty()`.